pkg/gardenlet/bootstrap/bootstrap.go

```diff
@@ -89,22 +87,11 @@ func RequestKubeconfigWithBootstrapClient(
 func DeleteBootstrapAuth(ctx context.Context, reader client.Reader, writer client.Writer, csrName, seedName string) error {
 	var username string
 
-	// try certificates v1 API first
 	csr := &certificatesv1.CertificateSigningRequest{}
-	if err := reader.Get(ctx, kutil.Key(csrName), csr); err == nil {
-		username = csr.Spec.Username
-	} else {
-		if !meta.IsNoMatchError(err) {
-			return err
-		}
-
-		// certificates v1 API not found, fall back to v1beta1
-		csrv1beta1 := &certificatesv1beta1.CertificateSigningRequest{}
-		if err := reader.Get(ctx, kutil.Key(csrName), csrv1beta1); err != nil {
-			return err
-		}
-		username = csrv1beta1.Spec.Username
+	if err := reader.Get(ctx, kutil.Key(csrName), csr); err != nil {
+		return err
 	}
+	username = csr.Spec.Username
```

Comment:
Same here, we don't need this variable anymore. Use `csr.Spec.Username` directly.